srcpkgs/gnome-text-editor/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'gnome-text-editor'
 pkgname=gnome-text-editor
-version=41.1
+version=42.0
 revision=1
 build_style=meson
-hostmakedepends="pkg-config gettext itstool glib-devel"
+hostmakedepends="pkg-config gettext itstool glib-devel desktop-file-utils"
```

Comment:
Why is this required?

Reply:
Because it the meson update afaict. Without it, meson errors:
```
Program glib-compile-schemas found: YES (/usr/bin/glib-compile-schemas)
Program gtk4-update-icon-cache found: NO
Program gtk-update-icon-cache found: YES (/usr/bin/gtk-update-icon-cache)
Program update-desktop-database found: NO
```